docker-compose/docker-compose.yml

```diff
@@ -207,6 +216,18 @@ services:
       - "5672:5672" # AMQP
       - "61616:61616" # OpenWire
       - "61613:61613" # STOMP
+  audit-storage:
+    image: quay.io/alfresco/alfresco-audit-storage:0.0.1-A7
+    mem_limit: 512m
+    environment:
+      SPRING_ACTIVEMQ_BROKERURL: failover:(nio://activemq:61616)?timeout=3000
+      AUDIT_ENTRYSTORAGE_OPENSEARCH_CONNECTOR_URI: http://elasticsearch:9200
+      AUDIT_ENTRYSTORAGE_OPENSEARCH_CONNECTOR_USERNAME: admin
+      AUDIT_ENTRYSTORAGE_OPENSEARCH_CONNECTOR_PASSWORD: admin
+      AUDIT_EVENTINGESTION_URI: activemq:topic:alfresco.repo.event2
+    depends_on:
+      - activemq
+      - elasticsearch
```

Comment:
@alxgomz we should probably move not-yet-released components to pre-release yml instead?

not sure if this new component should be treated as an optional one - so we could have a separate file like we did with solr6

Reply:
This component will be released "soon" The main goal of this PR is to get your opinion, adjust the config and be ready for the GA.

We have most of our components in the main docker-compose file so I think the audit storage should be there as well.

Reply:
pre-release imho, and we'll need to update the updatecli matrix as well.
Now, what worries me a bit here is to have that enabled by default. running this docker compose file will soon require more than 8GB and that's the base for the download trial. So maybe we should have an override file for this (and a related test job)?
Is there a frontend to that new feature already? Something trial users would really see a benefit for when testing Alfresco?

Reply:
I guess the frontend is kibana itself, customized dashboard are still under development, eventually we need to ship those as well